clang/lib/DPCT/AnalysisInfo.h

```diff
@@ -1934,6 +1934,9 @@ class DpctGlobalInfo {
   static bool useEnqueueBarrier() {
     return getUsingExtension(DPCPPExtensions::Ext_EnqueueBarrier);
```

Comment:
need a test case to see what if cxx_standard_library is off.

Reply:
added